src/app/chefs/ChefHome.jsx

```diff
@@ -20,6 +20,7 @@ class ChefHome extends Component {
       name: 'testchef', // may want to remove from here
       password: 'chefpass', // may want to remove from here
       imageUrl: 'chefimage.com', // may want to remove from here
+      id: 1,
```

Comment:
what does this do?

Reply:
I hard coded some data to test stuff, and I will need the chef id passed to the ChefSchedule component.